Dockerfile

```diff
@@ -1,24 +1,36 @@
-FROM debian:wheezy
+FROM debian:8.9
+MAINTAINER wekan
 
 # Declare Arguments
 ARG NODE_VERSION
 ARG METEOR_RELEASE
+ARG METEOR_EDGE
+ARG USE_EDGE
 ARG NPM_VERSION
+ARG FIBERS_VERSION
+ARG ARCHITECTURE
+ARG SRC_PATH
 
 # Set the environment variables (defaults where required)
-ENV ARCHITECTURE=linux-x64
-ENV BUILD_DEPS="wget curl bzip2 build-essential python git ca-certificates"
+ENV BUILD_DEPS="wget curl bzip2 build-essential python git ca-certificates gcc-4.9"
 ENV GOSU_VERSION=1.10
-ENV NODE_VERSION ${NODE_VERSION}
-ENV METEOR_RELEASE ${METEOR_RELEASE}
-ENV NPM_VERSION ${NPM_VERSION}
+ENV NODE_VERSION ${NODE_VERSION:-v4.8.4}
+ENV METEOR_RELEASE ${METEOR_RELEASE:-1.4.4.1}
+ENV USE_EDGE ${USE_EDGE:-false}
+ENV METEOR_EDGE ${METEOR_EDGE:-1.5-beta.17}
+ENV NPM_VERSION ${NPM_VERSION:-4.6.1}
+ENV FIBERS_VERSION ${FIBERS_VERSION:-1.0.15}
+ENV ARCHITECTURE ${ARCHITECTURE:-linux-x64}
+ENV SRC_PATH ${SRC_PATH:-./}
+ENV METEOR_PROFILE=100
+ENV METEOR_LOG=debug
 
 RUN \
     # Add non-root user wekan
-    useradd --user-group --system --home-dir /home/wekan wekan && \
+    useradd --user-group --system -m wekan && \
     \
     # OS dependencies
-    apt-get update -y && apt-get install -y --no-install-recommends ${BUILD_DEPS} && \
+    apt-get update -y && apt-get dist-upgrade -y && apt-get install -y --no-install-recommends ${BUILD_DEPS} && \
```

Comment:
Running `dist-upgrade` is against [best practices](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#run).